Tasks/Sepseleva/Task2/Task2.py

```diff
@@ -0,0 +1,85 @@
+import datetime
+
+time1 = {0:['ноль'], 1:['один','одной минуты','одна минута'], 2:['два','двух','две минуты' ],3:['три','трёх'], 
+    4:['четыре', 'четырех'], 5:['пять','пяти'], 6:['шесть','шести'], 7:['семь','семи'], 
+    8:['восемь','восьми'], 9:['девять','девяти'], 10:['десять','десяти'], 11:['одиннадцать', 'одиннадцати'], 12:['двенадцать','двенадцати'], 
+    13:['тринадцать','тринадцати'], 14:['четырнадцать','четырнадцати' ], 15:['пятнадцать','пятнадцати'], 
+    16:['шестнадцать','шестнадцати'], 17:['семнадцать','семнадцати' ], 18:['восемнадцать','восемнадцати' ],
+    19:['девятнадцать','девятнадцати' ], 20:['двадцать','двадцати'], 21:['двадцать одна минута','двадцати одной минуты'],
+    22:['двадцать две минуты','двадцати двух'], 23:['двадцать три','двадцати трех'], 24:['двадцать четыре','двадцати четырех'],
+    25:['двадцать пять','двадцати пяти'], 26:['двадцать шесть','двадцати шести'], 27:['двадцать семь','двадцати семи'], 
+    28:['двадцать восемь','двадцати восьми'], 29:['двадцать девять','двадцати девяти']}
```

Comment:
there are several not very obvious exceptions for some keys, it's ok in terms of the syntax, but not very clear